### PR TITLE
first pass at `create-next-app` integration and scaffolding

### PR DESCRIPTION
**Pull request by henripal**
This is to make concrete @with-heart proposal in Issue #3 

### steps I followed to create this:
1. created a `next` app with `create-next-app` in a `template` subfolder
2. Added web3 basics to the resulting template (`hardhat`, `waffle`, `etherjs`)
3. Added steps in `README` to get a basic app running using this template + `create-next-app` as suggested by @with-heart.

Note that steps 2. and 3. are only slight modifications from @dabit3's setup here: https://github.com/dabit3/full-stack-ethereum and are just intended to serve as placeholders.

### proposed usage
We would need to wrap the cli call in the first step in our own `create-dao` call. 
In short, the steps to reproduce the app and dependencies that are in the `\template` subfolder are as follows (these are also in the `README`):

1. Use the cli to generate the scaffold: `npx create-next-app --example=https://github.com/henripal/create-dao/tree/main/template` (replace this with the github address of the template in our own repo. parametrizable when we wrap this call in another cli)
2. Compile the existing contract: `npx hardhat compile` (right now this is just a fill-in contract: `template/contracts/Greeter.sol`)
3. Deploy a local network: `npx hardhat node` && Deploy your contract to the local network: `npx hardhat run scripts/sample-script.js --network localhost` Note the address of the contract.
4. Change the `greeterAddress` in `pages/index.js` to the one your contract was deployed to in 5. above.
5. Start your app: `npm run dev`

With this template in place, to get to `create-dao` we would need to:
1. Wrap the cli call in 1. above in our own cli
2. Change the /template folder to the template for the DAO we want (including the `contracts` subfolderr)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,4 +11,25 @@ The way [Nader](https://discord.com/channels/883478451850473483/8837055628508078
...
+
+5. Deploy your contract to the local network: `npx hardhat run scripts/sample-script.js --network localhost`
+Note the address of the contract.
+
+6. Change the `greeterAddress` in `pages/index.js` to the one your contract was deployed to in 5. above.
+
+7. Start your app: `npm run dev`
+
+(except the wrapping of `create-next-app` and a few modifications, this is mostly taken from https://dev.to/dabit3/the-complete-guide-to-full-stack-ethereum-development-3j13)
+
```
Comment (Dhaiwat10) [inline README.md:34]:
```suggestion

### How this template will be used on a livenet:

1. Run `npx create-dao <dao_name>`
2. Answer questions according to your preference.
3. `cd <dao_name>`
4. `yarn deploy:contract`
5. TBD..
```

**Review by Dhaiwat10 — APPROVED** [review]
This is a great starting point! Of course, there is a LONG way to go before this is what we need but I believe we can merge this in & iterate upon this. 

Thanks!

File: README.md (hunk 1 above)
Comment (Dhaiwat10) [reply README.md:34]: (maybe let's just add this at the bottom of the readme)

**Comment by henripal** [comment]
added your comment, thanks!

**Pull request merged**